Doc/source/release_notes.rst

```diff
@@ -42,6 +42,14 @@ New features
 now support reading from gzipped input files; filenames ending with
 ``.gz`` are assumed to be gzipped.
 
+:class:`~spacepy.pybats.ImfFile` has been updated with a more flexible
+quick-look plot capability. Variable naming has been unified to remove issues
+with temperature and density naming.
+
+:class:`~spacepy.pybats.bats.Bats2d` can now calculate parallel and
+perpendicular velocities with new `calc_uperp` and `calc_upar` methods.
```

Comment:
One change: make these `~.pybats.bats.Bats2d.calc_uperp` and `~.pybats.bats.Bats2d.calc_upar` and then they'll turn into links once we actually have docs built for them.